.github/workflows/main.yml

```diff
@@ -84,7 +84,6 @@ jobs:
       
       - name: Archive release build
         uses: actions/upload-artifact@v1
-        if: github.ref == 'ref/head/master'
         with:
           name: ${{ env.PROJECT_NAME }}-release_build-v${{ env.PROJECT_VERSION }}
           path: target/release/${{ env.PROJECT_NAME }}

```

Comment:
wouldn't this mean that after every push on dev a release version is build?

Reply:
no, because it is now just running on main (first change)